### ISSUE TITLE
libcpr static build option

### ISSUE THREAD

**Issue by Norman-Normandy** [libcpr/cpr#1094]
### Is your feature request related to a problem?

Would be nice to have a built in option for static builds. I.E libcpr.a on a posix OS. Or just have it build both static and dynamic.

### Possible Solution

Add a static library option

**Comment by Norman-Normandy** [comment]
To answer my own question, to build a static library, one can do:

$ git clone https://github.com/libcpr/cpr.git
$ cd cpr && mkdir build && cd build
$ cmake .. -DCPR_USE_SYSTEM_CURL=ON -DBUILD_SHARED_LIBS=OFF
$ cmake --build . --parallel
$ sudo cmake --install . 

From from this [issue](https://github.com/libcpr/cpr/issues/956)

It would help new users if it was mentioned somewhere.

**Comment by COM8** [comment]
Yes, agree. Would you like to create a PR for a bit of documentation in the README.md?

**Comment by Norman-Normandy** [comment]
> Yes, agree. Would you like to create a PR for a bit of documentation in the README.md?

Sure. Created a PR request #1095

**Comment by JoseYUE** [comment]
Could I build a static cpr lib via Fetch_Content?
